go/store/types/blob.go

```diff
@@ -464,3 +475,23 @@ func readBlob(ctx context.Context, r io.Reader, vrw ValueReadWriter) (Blob, erro
 
 	return newBlob(seq), nil
 }
+
+func (Blob) MarshalToKind(NomsKind) MarshalCallback {
```

Comment:
In the code that's currently present, we're doing nil checks, but from just checking now it looks like we're forgetting to do some nil checks in some places, so right now we definitely have some places that may unexpectedly panic instead of failing gracefully! Great catch 😄